src/components/BoilerplateLayout/BoilerplateLayout.js

```diff
@@ -147,7 +150,10 @@ const BoilerplateLayout = ({ children }) => {
         <LayoutSystemProvider {...layoutConfig}>
           <Layout
             header={
-              <LayoutHeader left={<Back />} center={<Logo height="40" />} />
+              <LayoutHeader
+                left={isVippsCallbackPage ? null : <Back />}
```

Comment:
Please, move this computed property outside of the JSX-template (senseteq-coding-standards, rules 5+8 together)